src/GridFS/Bucket.php

```diff
@@ -100,6 +104,10 @@ public function __construct(Manager $manager, $databaseName, array $options = []
             throw new InvalidArgumentException(sprintf('Expected "chunkSizeBytes" option to be >= 1, %d given', $options['chunkSizeBytes']));
         }
 
+        if (isset($options['disableMD5']) && ! is_bool($options['disableMD5'])) {
+            throw InvalidArgumentException::invalidType('"disableMD5" option', $options['disableMD5'], 'boolean');
+        }
```

Comment:
Can we add some tests for this? Should just require adding the following to `BucketFunctionalTest::provideInvalidConstructorOptions()`:

```
foreach ($this->getInvalidBooleanValues() as $value) {
    $options[][] = ['disableMD5' => $value];
}
```